pkgs/development/tools/pex/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, python3Packages }:
+with python3Packages; buildPythonApplication rec {
+    name = "${pname}-${version}";
+    pname  = "pex";
+    version = "1.4.5";
+
+    src = fetchPypi {
+      sha256 = "04s9qvx87ngfs3m91qsrmk0ll16s0ldrvlxjbg4y1ic4bgsjq3hj";
+      inherit pname version;
+    };
+
+    propagatedBuildInputs = [ requests wheel ];
+
+    prePatch = ''
+      substituteInPlace setup.py --replace 'SETUPTOOLS_REQUIREMENT,' '"setuptools",'
+      substituteInPlace setup.py --replace 'WHEEL_REQUIREMENT,' '"wheel",'
+    '';
+    patches = [ ./set-source-date-epoch.patch ];
+
+    # A few more dependencies I don't want to handle right now...
+    doCheck = false;
```

Comment:
Did you check whether these dependencies have been packaged in the meantime?

Reply:
No, I just copied the existing definition. Removing the line seems to work fine -- though no tests are run.

Please let me know how you like to see changes made in response to review comments: as follow-up commits, or by me immediately fixing the broken commit and force pushing?